odd_collector/adapters/superset/mappers/dashboards.py

```diff
@@ -0,0 +1,30 @@
+from ..domain.dataset import Dataset
+from ..domain.dashboard import Dashboard
+from typing import List
+from odd_collector.adapters.superset.plugin.plugin import SupersetGenerator
+from odd_models.models import DataEntity, DataEntityType, DataConsumer
+
+
+def create_dashboards_entities(
```

Comment:
Rename to map_dashboard, that function should take only one Dashboard, i.e:

```python 
def map_dashboard(
    oddrn_generator: SupersetGenerator,
    datasets: List[Dataset],
    dashboard: Dashboard
) -> DataEntity:
    ...
```
For loop should be on the caller side (i.e on adapter)